sycl/include/sycl/kernel_bundle.hpp

```diff
@@ -743,6 +808,138 @@ build(const kernel_bundle<bundle_state::input> &InputBundle,
   return build(InputBundle, InputBundle.get_devices(), PropList);
 }
 
+namespace ext::oneapi::experimental {
+
+/////////////////////////
+// PropertyT syclex::build_options
```

Comment:
Could you please cleanup comments from syclex or remove comments which just  duplicate the name of the definition.

Reply:
Do you mean remove the string "syclex::" from the comments? I think it's useful. This is kernel_bundle.hpp, most of this file is in the `sycl` or `sycl::detail` namespace. Only these new extensions are in `sycl::ext::oneapi::experimental` so it's useful to remind the reader of that. And `syclex::` reads a lot nicer than `sycl::ext::oneapi::experimental`

And this isn't just the comment for the struct build_options below, it's the comment introducing the PropertyT syclex::build_options, which includes the struct, the property key templates, and runtime/compile time declarations. Everything between this comment and the next one.